src/settings/index.ts

```diff
@@ -110,6 +110,8 @@ export const MINIMUM_AUDITORIUM_ROWS_COUNT = 5;
 export const MAXIMUM_AUDITORIUM_ROWS_COUNT = 5;
 // playa is 4000x4000 pixels, Burning Seed paddock is 2000x2000
 
+export const DEFAULT_TEXTAREA_ROWS = 4;
+
 export const ACCEPTED_IMAGE_TYPES =
   "image/png,image/x-png,image/gif,image/jpg,image/jpeg,image/tiff,image/bmp,image/gif,image/webp";
```

Comment:
In order to reduce the clutter in this `index.ts` file, we should try to group these settings and put them into separate files like in the `export *` section above.

Maybe `adminDesignSettings.ts` or whatever that makes sense and follows the local naming convention.

Reply:
created `adminDesignSettings.ts`: https://github.com/sparkletown/sparkle/pull/2868/commits/ba937c968da803ade57ed5627ca35f3baeade6a0